### PR TITLE
Add APIs to restart a service.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/constants/EventTypes.js [hunk 1]
```diff
@@ -69,6 +69,8 @@ let EventTypes = {};
   'MARATHON_SERVICE_DELETE_SUCCESS',
   'MARATHON_SERVICE_EDIT_ERROR',
   'MARATHON_SERVICE_EDIT_SUCCESS',
+  'MARATHON_SERVICE_RESTART_ERROR',
+  'MARATHON_SERVICE_RESTART_SUCCESS',
```
Comment (kennyt) [inline src/js/constants/EventTypes.js]: should we add these to `StoreConfig`?

File: src/js/events/MarathonActions.js [hunk 2]
```diff
@@ -38,10 +40,14 @@ var Config = require('../config/Config');
 import MarathonUtil from '../utils/MarathonUtil';
 import Util from '../utils/Util';
 
+function buildURI(path) {
+  return `${Config.rootUrl}${Config.marathonAPIPrefix}${path}`;
+}
```
Comment (kennyt) [inline src/js/events/MarathonActions.js]: good stuff

File: src/js/constants/EventTypes.js (hunk 1 above)
Comment (rcorral) [inline src/js/constants/EventTypes.js]: Yup, done!

**Comment by mlunoe** [comment]
Awesome! This looks good! Could we get a screenshot of the integration tests passing?

**Comment by rcorral** [comment]
I'm not adding or changing any View layer bits. I don't think that's necessary for this guy. Next PR, I'll do that!

**Pull request merged**